site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -14,12 +14,22 @@ Not familiar with Cypress? Here is some documentation that will help you get sta
 
 ## Quick Overview on How to Run E2E Testing
 
+### In Local Development Machine
 1.  Run a local server instance by initiating `make run` to the mattermost-server folder. Confirm that the instance has started successfully.
-   - Run `make test-data` to preload your server instance with initial seed data.  Generated data such as users are typically used for logging, etc.
-   - Each test case should handle the required system or account settings, but in case you encountered some unexpected error while testing, you may want to run the server with default config based on `mattermost-server/config/default.json`).
-2.  Change directory to `mattermost-webapp/e2e`, install npm dependencies by `npm i` and initiate `npm run cypress:run` in the command line. This will start full E2E testing. To run partial testing, you may initiate `npm run cypress:open` in the command line which will open its desktop app.  From there, you may select particular tests you would like to run.
+    - Run `make test-data` to preload your server instance with initial seed data.
+    - Each test case should handle the required system or user settings, but in case you encountered some unexpected error while testing, you may want to run the server with default config based on `mattermost-server/config/default.json`).
```

Comment:
```suggestion
- Each test case should handle the required system or user settings, but in case you encounter an unexpected error while testing, you may want to run the server with default config based on `mattermost-server/config/default.json`.
```